fabric-biomes-v1/src/main/java/net/fabricmc/fabric/mixin/biome/MixinMultiNoiseBiomeSource.java

```diff
@@ -20,33 +20,45 @@
 import java.util.List;
 import java.util.function.Supplier;
 
+import com.google.common.collect.ImmutableList;
 import com.mojang.datafixers.util.Pair;
 import org.spongepowered.asm.mixin.Mixin;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
 
+import net.minecraft.util.registry.Registry;
 import net.minecraft.world.biome.Biome;
 import net.minecraft.world.biome.source.MultiNoiseBiomeSource;
 
 import net.fabricmc.fabric.impl.biome.InternalBiomeData;
 
-@Mixin(MultiNoiseBiomeSource.class)
+/**
+ * This Mixin is responsible for adding mod-biomes to the NETHER preset in the MultiNoiseBiomeSource.
+ */
+@Mixin(MultiNoiseBiomeSource.Preset.class)
 public class MixinMultiNoiseBiomeSource {
```

Comment:
Just gonna mention that I like my approach [in my fork](https://github.com/CheaterCodes/fabric/blob/biome-api/fabric-biomes-v1/src/main/java/net/fabricmc/fabric/mixin/biome/MultiNoiseBiomeSourcePresetMixin.java) better, but this is probably fine too.

Reply:
That might actually be better. I'll investigate adopting that approach.

Reply:
Adopted your approach